src/System Application/App/Cryptography Management/src/dotnet.al

```diff
@@ -47,12 +47,10 @@ dotnet
         type("System.Security.Cryptography.OidCollection"; OidCollection) { }
         type("System.Security.Cryptography.Oid"; Oid) { }
         type("System.Security.Cryptography.AesCryptoServiceProvider"; "Cryptography.AesCryptoServiceProvider") { }
-
     }
-
+    
```

Comment:
Could you revert this change? There is none, so let's not pollute this PR with unnecessary object changes.